python/lsst/meas/algorithms/loadReferenceObjects.py

```diff
@@ -264,13 +264,18 @@ class ReferenceObjectLoader:
         An iterable object of data IDs that point to reference catalogs.
     refCats : iterable of `lsst.daf.butler.DeferredDatasetHandle`
         Handles to load refCats on demand.
+    name : `str`
+        The name of the refcat that this object will load. This name is used
+        for applying colorterms, for example.
+    config : `lsst.pex.config.Config`
```

Comment:
Doesn't this need to be an instance of `LoadReferenceObjectsConfig` (or at least a subclass)?

Reply:
Good catch, yes!